package.json

```diff
@@ -201,7 +201,7 @@
     "tslint-etc": "1.13.10",
     "tslint-no-toplevel-property-access": "0.0.2",
     "tslint-no-unused-expression-chai": "0.0.3",
-    "typescript": "~4.2.0",
+    "typescript": "~4.3.0",
```

Comment:
We need to be backwards compatible  with TS 4.2 or this is a breaking change. 🤔

Reply:
This change was only required to **emit** different `.d.ts`. It shouldn't break TS 4.2 consumers because they only consume the emitted `.d.ts`.

Take a look at the 4.2 playground [here](https://www.typescriptlang.org/play?ts=4.2.3#code/PTAEHMFMDtIJwIYBdIBNQFcDOBLa5QAVAZVABYA6AZgFgAoSADwAcB7OJUAbwF8BueqkgBjADYI4kCKNYAjBKO71QK0HhRwAZgmFTiATwC2s1qIDCraFiRwMwpOyV1VL0JISpLo-aDlYAXKBYRiaiAs6qPPRRdEJiElLCltZBRgDqiMzM8IFcyqrBhoFI+tmsmqAGxqYUfuH89ExsHKBx4pKg8VhYoACCTjFN7JxtCZ3i3aAAQgONLMOtIu2JEz1mswzzLaMdXT0AIk4uANqFGQhZ8BSFALqBABQAlKAAvAB8mNBCmnho4S7MOCsZiBaAYYzwaL0dTwbS6UBpHBIAAWAEloBpgQAeQgfPIRFTHKqhWqyLB3UBPV4fQj1ej0ECgFDWPAEJJCel0JJWEavUCwADuoH2T05jJROB6AssAHJOLIpAhujhwNAELJRFI8ERSJQAEygJVqTjCBDQOWfIRwaxm9AoqTBDEIRhqaCgAAGe2F7qZyOQah6kiwbCsOA1Uk0jgUMgFrN9koZYAAAkgsABaJoiJAZuBAuBzZom5KcJW4VXqzWBREo9GY5hYjBfSA-WCoD4vVpisCe1begNBJA4USKABu8B8hjsyMwWArUgFyJgxs6rEMkB6Dl89rgoEBwPgg-XvtYGHAyIoOv7AEcMEipOBWKx0P6FXHtBhRNmHJrEBjQAulyRFc1w3VhdwkXB8AAGlAWQ8FQVkYMgJBhALBZuRSMEin5cEFR3DtUAoPdmHoIA). It works as expected. 

We can also check the same code with TS 4.3 [here](https://www.typescriptlang.org/play?ts=4.3.5#code/PTAEHMFMDtIJwIYBdIBNQFcDOBLa5QAVAZVABYA6AZgFgAoSADwAcB7OJUAbwF8BueqkgBjADYI4kCKNYAjBKO71QK0HhRwAZgmFTiATwC2s1qIDCraFiRwMwpOyV1VL0JISpLo-aDlYAXKBYRiaiAs6qPPRRdEJiElLCltZBRgDqiMzM8IFcyqrBhoFI+tmsmqAGxqYUfuH89ExsHKBx4pKg8VhYoACCTjFN7JxtCZ3i3aAAQgONLMOtIu2JEz1mswzzLaMdXT0AIk4uANqFGQhZ8BSFALqBABQAlKAAvAB8mNBCmnho4S7MOCsZiBaAYYzwaL0dTwbS6UBpHBIAAWAEloBpgQAeQgfPIRFTHKqhWqyLB3UBPV4fQj1ej0ECgFDWPAEJJCel0JJWEavUCwADuoH2T05jORrHYCH0AEImcicD0BewANZYObNTjclIIbo4cDQBCyUSQQKIlHozHMLEYL6QH6wVAfF6tTlDFrazhgor88GyeB81AUQHA+hAA) and verify that with this version this property can be understood in full and used as expected, allowing Rx observable to be treated as `InteropObservable`.

The only downside to the current state of this PR is that with TS 4.2 people might get errors because of this pattern if they dont use `skipLibCheck`. We can suppress it with `/** @ts-ignore */` though and that should make it compatible with all of the requirements.

Let me know what do you think and I will followup with any changes/answers as needed.